infrastructure/equinix-metal/main.tf

```diff
@@ -4,6 +4,10 @@ terraform {
       source  = "equinix/equinix"
       version = "1.13.0"
     }
+    random = {
+      source  = "hashicorp/random"
```

Comment:
what about static token?

Reply:
Yes a static token added as a GitHub secret would work 👍

Reply:
Check this one https://registry.terraform.io/modules/scholzj/kubeadm-token/random/latest

Reply:
okay some specific env variable will contain the token?

Reply:
It looks good.  

Also on the Hashicorp random provider I was wrong its MPL not BSL but it might be relicensed at some point so probably best not to rely on it.

Reply:
sure, what about `env.BOOTSTRAP_K3s_TOKEN` ?

Reply:
Can we use `env.K3s_AGENT_TOKEN` to match k3s?

Reply:
sure

Reply:
I was thinking that we can create a simple variable `k3s_agent_token`

then in the githubactions we can use `TF_VAR_k3s_agent_token=""`

Reply:
> then in the githubactions we can use TF_VAR_k3s_agent_token=""

Yes that's nice we do the same for the Equinix auth token

Reply:
Done